test/post.test.js

```diff
@@ -93,6 +93,27 @@ describe('Post data class tests', () => {
     expect(() => createPostWithFeed(feed)).not.toThrow();
   });
 
+  test('Post constructor should throw if invalid dates is passed', () => {
```

Comment:
I think we should change this to:
`test('Post constructor should throw if a string or date is not passed`

What do you think? Since it will fail if either of the values are not a string/date

Reply:
I think that makes sense, ill fix that.